.goreleaser.yml

```diff
@@ -0,0 +1,49 @@
+before:
+  hooks:
+    - go test ./...
+
+builds:
+  - id: signable
+    mod_timestamp: '{{ .CommitTimestamp }}'
+    targets:
+      - darwin_amd64
+      - windows_386
```

Comment:
Are we signing the Windows 386 builds? I didn't think we were, and so packagespec does not at the moment, but if that's wrong I need to change it ASAP!

Reply:
Yes, we are signing win 386 builds (as well as amd64 + darwin amd64). I'll go through the other products this week to make sure they're doing this as well